protocols/discord.py

```diff
@@ -227,6 +227,10 @@ def _burst_new_client(self, guild, member, pylink_netobj):
             log.debug('(%s) Not bursting user %s as their data is not ready yet', self.protocol.name, member)
             return
 
+        if not pylink_netobj.serverdata.get('join_offline_users', True) and \
+                member.user.presence in [DiscordStatus.OFFLINE, DiscordStatus.INVISIBLE, UNSET]:
```

Comment:
Could we move this into a helper function like `_should_burst_user()`? I believe it's possible to invert the logic as follows:

```python
return pylink_netobj.serverdata.get('join_offline_users', True) or \
    member.user.presence not in [DiscordStatus.OFFLINE, DiscordStatus.INVISIBLE, UNSET]
```